src/sources/dnstap/mod.rs

```diff
@@ -30,8 +30,9 @@ use vector_core::config::{LegacyKey, LogNamespace};
 #[configurable_component(source("dnstap"))]
 #[derive(Clone, Debug)]
 pub struct DnstapConfig {
-    /// Maximum length, in bytes, that a frame can be.
+    /// Max DNSTAP frame length that the dnstap source can handle.
```

Comment:
🤔  the wording of "that the dnstap source can handle" puzzles me a bit. A maximum value that a source can handle seems like it isn't something that should be configurable. Which makes me question the wording. Maybe change "can" -> "will" ? And if the frame is larger than this, what happens, is it dropped?

Reply:
Good point, maybe more like:

```
/// Max DNSTAP frame length that the dnstap source will accept.
```